assets.RealTimeFeedIT tests are failing

#### Description

After the dependency upgrades in #8919 one or more tests under [asset][1] fail in Java 11 ([sponge][s11]).

Based on the errors:

```
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 4.716 s <<< FAILURE! - in com.example.asset.RealTimeFeedIT
[ERROR] com.example.asset.RealTimeFeedIT  Time elapsed: 4.686 s  <<< ERROR!
com.google.api.gax.rpc.AlreadyExistsException: io.grpc.StatusRuntimeException: ALREADY_EXISTS: Resource already exists in the project (resource=topicId).
Caused by: io.grpc.StatusRuntimeException: ALREADY_EXISTS: Resource already exists in the project (resource=topicId).

[ERROR] Errors: 
[ERROR]   RealTimeFeedIT » AlreadyExists io.grpc.StatusRuntimeException: ALREADY_EXISTS:...
[ERROR] Tests run: 20, Failures: 0, Errors: 1, Skipped: 1
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-failsafe-plugin:3.0.0-M5:verify (default) on project cloudasset-snippets: There are test failures.
```

the problem is that creating a test topic "topicId" fails because there is already a topic with this name.

[1]: https://github.com/renovate-bot/java-docs-samples/tree/7ef3ac48409470194438b459e86ab054b975a96c/asset
[s11]: https://btx-internal.corp.google.com/invocations/b28f9f6f-ea75-4c62-bd5d-48572c227087/targets/cloud-devrel%2Fjava%2Fjava-docs-samples%2Fjava11%2Fpresubmit;config=default/log

--- a/asset/src/test/java/com/example/asset/RealTimeFeedIT.java
+++ b/asset/src/test/java/com/example/asset/RealTimeFeedIT.java
@@ -32,7 +32,6 @@ import org.junit.AfterClass;
 import org.junit.Before;
 import org.junit.BeforeClass;
 import org.junit.FixMethodOrder;
-import org.junit.Ignore;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
@@ -42,7 +41,6 @@ import org.junit.runners.MethodSorters;
 @RunWith(JUnit4.class)
 @SuppressWarnings("checkstyle:abbreviationaswordinname")
 @FixMethodOrder(MethodSorters.NAME_ASCENDING)
-@Ignore("TODO: Fix https://github.com/GoogleCloudPlatform/java-docs-samples/issues/8963")
 public class RealTimeFeedIT {
   private static final String topicId = "topicId";
   private static final String feedId = UUID.randomUUID().toString();
@@ -63,14 +61,16 @@ public class RealTimeFeedIT {
 
   @BeforeClass
   public static void createTopic() throws Exception {
-    TopicAdminClient topicAdminClient = TopicAdminClient.create();
-    topicAdminClient.createTopic(topicName);
+    try (TopicAdminClient topicAdminClient = TopicAdminClient.create()) {
+      topicAdminClient.createTopic(topicName);
+    }
   }
 
   @AfterClass
   public static void deleteTopic() throws Exception {
-    TopicAdminClient topicAdminClient = TopicAdminClient.create();
-    topicAdminClient.deleteTopic(topicName);
+    try (TopicAdminClient topicAdminClient = TopicAdminClient.create()) {
+      topicAdminClient.deleteTopic(topicName);
+    }
   }
 
   @Before

